python/lsst/meas/deblender/deblend.py

```diff
@@ -121,6 +121,14 @@ class SourceDeblendConfig(pexConf.Config):
         doc=("Mask planes with the corresponding limit on the fraction of masked pixels. "
              "Sources violating this limit will not be deblended."),
     )
+    weightTemplates = pexConf.Field(dtype=bool, default=False,
+                                    doc=("If true, a least-squares fit of the templates will be done"))
+    removeDegenerateTemplates = pexConf.Field(dtype=bool, default=False,
+                                              doc=("Try to remove similar tempates?"))
+    maxTempDotProd = pexConf.Field(dtype=float, default=0.5,
+                                   doc=("Threshold on the dot product between two templates that "
+                                        "determines whether they are degenerate.  Pairs with values "
+                                        "larger than this will remove one of the templates."))
```

Comment:
English is a bit off in this sentence...but if you really prefer the brevity, I can live with it!